databases/week2/creatingDatabase/week2HWscript.sql

```diff
@@ -0,0 +1,102 @@
+CREATE TABLE `customers` (
+  `customerID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `firstName` varchar(50) NOT NULL,
+  `lastName` varchar(50) NOT NULL,
+  `email` varchar(50) NOT NULL UNIQUE,
+  `phone` varchar(20) NOT NULL,
+  `created_at` TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (`customerID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `orders` (
+  `orderID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `order_date` DATE NOT NULL,
+  PRIMARY KEY (`orderID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `customerOrder` (
+  `orderID` int(10) unsigned NOT NULL,
+  `customerID` int(10) unsigned NOT NULL,
+  `order_status` ENUM('pending','shipped','delivered','cancelled') NOT NULL DEFAULT 'pending',
+  PRIMARY KEY (`orderID`),
+  CONSTRAINT `customerOrder_customerID` FOREIGN KEY (`customerID`) REFERENCES `customers` (`customerID`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `customerOrder_orderID` FOREIGN KEY (`orderID`) REFERENCES `orders` (`orderID`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `country` (
+  `countryID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `countryName` varchar(50) NOT NULL,
+  PRIMARY KEY (`countryID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `brand` (
+  `brandID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `brandName` varchar(50) NOT NULL,
+  `countryID` int(10) unsigned NOT NULL,
+  PRIMARY KEY (`brandID`),
+ CONSTRAINT `brand_countryID` FOREIGN KEY (`countryID`) REFERENCES `country` (`countryID`) ON DELETE CASCADE ON UPDATE CASCADE
+  -- FOREIGN KEY (countryID) REFERENCES country(countryID)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `item` (
+  `itemID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `itemName` varchar(50) NOT NULL,
+  `itemPrice` DECIMAL(10,2) NOT NULL,
```

Comment:
line 44, great u use tan appropriate datatype

Reply:
thank you